contrib/tools/dummysigner/README.md

```diff
@@ -15,6 +15,28 @@ the [bip32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) form
 cargo run -- <xpriv> <xpriv> ... 
 ```
 
+The dummysigner handles revault descriptors
+
+```
+cargo run -- --conf  <config_paht> 
+```
+
+with config:
```

Comment:
nit: instead of putting the config here, maybe link it to `contrib/tools/dummysigner/examples/examples_cfg.toml`